cardano_node_tests/tests/test_native_tokens.py

```diff
@@ -1886,6 +1888,84 @@ def test_transfer_no_ada(
                 else:
                     pytest.fail(msg)
 
+    @allure.link(helpers.get_vcs_link())
+    @hypothesis.given(
+        token_amount=st.integers(min_value=20_000_001, max_value=MAX_TOKEN_AMOUNT - 1)
```

Comment:
Shouldn't `MAX_TOKEN_AMOUNT = (2**64) - 1` then? Otherwise `MAX_TOKEN_AMOUNT` is not "max token amount", but "max token amount" + 1, no?

Reply:
I don't even remember why I put -1, I corrected to just MAX_TOKEN_AMOUNT